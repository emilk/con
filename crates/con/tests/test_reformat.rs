#[test]
fn test_reformat() {
    let input = r#"
        // This comment is outside the outermost object.
        {
            // This comment proceeds the first key-value pair.
            key: "value" //Suffix comment


            // Comment about the second key-value pair.
            key:
            // Very weird comment
            42,
        }
    "#;

    let formatted = con::reformat(input, &Default::default()).unwrap();
    insta::assert_snapshot!(formatted);
}
