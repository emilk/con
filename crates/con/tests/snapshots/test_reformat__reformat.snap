---
source: crates/con/tests/test_reformat.rs
expression: formatted
---
// This comment is outside the outermost object.
// This comment proceeds the first key-value pair.
key: true // Suffix comment

// Comment about the second key-value pair.
// Very weird comment
key: null
// TODO: add more test cases here
